ci/deps/actions-38-db.yaml

```diff
@@ -25,11 +25,10 @@ dependencies:
   - flask
   - nomkl
   - numexpr
-  - numpy=1.17.*
+  - numpy=1.18
```

Comment:
do we need this as 1.18.*?

Reply:
I believe 1.18.* and 1.18 are equivalent. They both resolve to the latest 1.18 version.